polar-core/src/resource_block.rs

```diff
@@ -1518,4 +1521,54 @@ mod tests {
 
         Ok(())
     }
+
+    // Test creation of rule types for actor roles
+    //   - has_role created because at least one resource block has roles declared
+    #[test]
+    fn test_create_resource_specific_rule_types_actor_roles() -> core::result::Result<(), PolarError>
+    {
+        let policy = r#"
+            actor User {}
```

Comment:
what's the point of having the `actor User {}` block in this test?

Reply:
this arrived via me copying in the policy from the original issue. I can ✂️ it out as only the `actor Team` block is crucial to testing the `has_role` rule type creation here.

Reply:
trimmed. thanks for the sohut.